src/realtime-callbacks.js

```diff
@@ -39,9 +39,10 @@ let _realCallbackKey;
 // each is an object with keys [runAt, func, params, key].
 const _callbackList = [];
 
+import logger from '../src/logger';
+
 // var debuglog = console.log.bind(console);
```

Comment:
should fix the debuglog too, and please add line 44's blank line back

Reply:
This is fixed